graphlearn/python/c/py_wrapper.h

```diff
@@ -678,6 +678,7 @@ PyObject* get_dag_value(GetDagValuesResponse* res,
   auto t = res->GetValue(node_id, key);
   auto values = std::get<0>(t);
   if (values == nullptr) {
+    Py_INCREF(Py_None); // 增加None的引用计数，以防止Python回收它
     return Py_None;
```

Comment:
You can also replace lines 681-682 with Py_RETURN_NONE for a more concise code.

Reply:
right, i will update code now